docs/framework/app-domains/how-to-install-an-assembly-into-the-gac.md

```diff
@@ -1,79 +1,55 @@
 ---
-title: "How to: Install an Assembly into the Global Assembly Cache"
-ms.date: "03/30/2017"
-helpviewer_keywords: 
+title: Install an assembly into the GAC
+ms.date: 09/20/2018
+helpviewer_keywords:
   - "assemblies [.NET Framework], global assembly cache"
   - "Gacutil.exe"
   - "strong-named assemblies, global assembly cache"
   - "global assembly cache, installing assemblies"
   - "Global Assembly Cache tool"
+  - "windows installer, global assembly cache"
 ms.assetid: a7e6f091-d02c-49ba-b736-7295cb0eb743
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# How to: Install an Assembly into the Global Assembly Cache
-There are two ways to install a strong-named assembly into the global assembly cache (GAC):  
-  
+# How to: Install an assembly into the global assembly cache
+
+There are two ways to install a strong-named assembly into the global assembly cache (GAC).
+
+## Windows Installer
+
+[Windows Installer](/windows/desktop/Msi/installation-of-assemblies-to-the-global-assembly-cache), the Windows installation engine, is the recommended and most common way to add assemblies to the global assembly cache. Windows Installer provides reference counting of assemblies in the global assembly cache, plus other benefits. You can use the [WiX Toolset extension for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension) to create an installer package for Windows Installer.
+
+## Global assembly cache tool
+
+You can use [Global assembly cache tool (gacutil.exe)](../tools/gacutil-exe-gac-tool.md) to add strong-named assemblies to the global assembly cache and to view the contents of the global assembly cache.
+
+   > [!NOTE]
+   > Gacutil.exe is only for development purposes and should not be used to install production assemblies into the global assembly cache.
+
+The syntax for gacutil is as follows:
+
+```shell
+gacutil -i <assembly name>
+```
+
+In this command, *assembly name* is the name of the assembly to install in the global assembly cache.
+
+The following example installs an assembly with the file name `hello.dll` into the global assembly cache.
+
+```shell
+gacutil -i hello.dll
+```
+
 > [!IMPORTANT]
```

Comment:
Side-by-side notes (Important followed by Note) is not optimal. This also is important -- without a strong-named assembly, there's no point in reading further. I'd move this immediately after the article's first paragraph.